core/api/src/servers/graphql-admin-server.ts

```diff
@@ -112,9 +112,11 @@ export async function startApolloServerForAdminSchema() {
   })
 }
 
-setupMongoConnection()
-  .then(async () => {
-    activateLndHealthCheck()
-    await startApolloServerForAdminSchema()
-  })
-  .catch((err) => graphqlLogger.error(err, "server error"))
+if (require.main === module) {
+  setupMongoConnection()
+    .then(async () => {
+      activateLndHealthCheck()
+      await startApolloServerForAdminSchema()
+    })
+    .catch((err) => graphqlLogger.error(err, "server error"))
+}
```

Comment:
```suggestion
```

Reply:
I think we. can just removed it

the reason I remove the wrapper around `if (require.main === module) {` initially was to make it compatible with `bun`.